Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -102,7 +102,7 @@ public override void OnPostSceneQuery()
                 return;
             }
 
-            if (Result?.CurrentPointerTarget != null)
+            if (Result?.CurrentPointerTarget != null && closestProximityTouchable != null)
```

Comment:
@MenelvagorMilsom this is the actual fix :-) the other changes in this PR are to fix a playmode test, remove some extra logging, and then add a playmode test that repros this issue so we don't hit it again.

Reply:
Makes sense to me! Thanks for catching this.